wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Publicize: add percentage data bar

Fixes #10746 

To test:

- On a site with social connections, go to Insights > Publicize.
  - en.blog at least has Twitter.
- Be amazed:
<img width="350" alt="publicize_data_bar" src="https://user-images.githubusercontent.com/1816888/51340000-147b3900-1a4b-11e9-8427-715649a94706.png">


**NOTE**: I don't have a site with multiple social connections, but @rachelmcr was kind enough to verify on one of her sites that the list is in fact sorted by total followers. So the rows after the first will correctly have partial bars related to the first.
